ui/src/utils/time.ts

```diff
@@ -32,9 +34,35 @@ enum Unit {
     Second = 's',
 }
 
+// tslint:disable-next-line:cyclomatic-complexity mccabe-complexity
 export const parseRelativeTime = (value: string, divide: 'endOf' | 'startOf', nowDate = moment()): Success | Failure => {
-    if (isValidDate(value)) {
-        return success(asDate(value));
+    for (const format of ['YYYY-MM-DD HH:mm', 'YYYY-MM-DD', 'YYYY-MM-DD[T]HH:mm:ssZ']) {
```

Comment:
I think this should only support `L` and `L LT` from https://momentjs.com/docs/?%2Fdisplaying%2Fformat%2F#/displaying/ so that the date can be formatted in the user defined date locale. `YYYY-MM-DD` can be supported as a fallback. `L` or `L LT` should be used below when passing it as `localized` to the success function.